CHANGES.txt

```diff
@@ -37,21 +37,5 @@ Changes
 Fixes
 =====
 
-- Fixed an issue that caused an UnsupportedFeatureException to be thrown for
-  queries with a ``WHERE`` clause which contains an equality comparison that
-  references a table column in both sides. E.g.::
-
-    SELECT * FROM t
-    WHERE t.i = abs(t.i)
-
 - Handle ``STRING_ARRAY`` as argument type for user-defined functions correctly
```

Comment:
This fix has not been backported to 3.0 branch, only to 2.3 meaning this will only be in the next release 3.0.3.